packages/http-server/src/http-server.ts

```diff
@@ -71,8 +100,12 @@ export class HttpServer {
   private _listening = false;
   private _protocol: HttpProtocol;
   private _address: string | AddressInfo;
-  private requestListener: RequestListener;
-  readonly server: http.Server | https.Server;
+  private requestListener: RequestListener | Http2RequestListener;
+  readonly server:
```

Comment:
Can we keep the server as a private property please?

Reply:
AFAIK the access modifiers weren't modified and making this change would be a breaking change. Could you please clarify?

Reply:
Sorry, I think I got confused because of how GitHub rendered this change. You are right, we should not be modifying access modifiers in this PR.